database/sqlite/sqlite_functions.c

```diff
@@ -62,7 +61,7 @@ const char *database_config[] = {
     "CREATE INDEX IF NOT EXISTS health_log_d_ind_2 ON health_log_detail (global_id);",
     "CREATE INDEX IF NOT EXISTS health_log_d_ind_3 ON health_log_detail (transition_id);",
     "CREATE INDEX IF NOT EXISTS health_log_d_ind_4 ON health_log_detail (health_log_id);",
-    //TODO more indexes
+    "CREATE INDEX IF NOT EXISTS health_log_d_ind_5 ON health_log_detail (when_key);",
```

Comment:
Planner indicates index on `when_key` is not used

Reply:
Ok, removed, thanks!